### DISCUSSION TITLE
[Question][Hw0-3] Language of Writing

### DISCUSSION THREAD

**Discussion by Lee-Janggun** [prosyslab-classroom/cs424-program-reasoning#383] [Q&A]
The instructions say that "Write in Korean if you are a native Korean speaker, in English otherwise."

Does that mean that I am forced to write Korean if I can, even if I prefer to write it in English? 

I just wanted to double check because I haven't seen a course where writing in Korean was forced.

Thank you!

**Comment by KihongHeo** [comment]
Hi Janggun,

The requirement is mandatory. 

As an educational institution, KAIST mainly has two goals:
1. educate students to become world-class researchers (for all students)
2. educate students to become leaders (e.g., faculty, CEO, politician, etc) in this country (mostly for Korean students)

This homework is a practice for Goal 2.
KAIST students should be able to clearly deliver their idea in Korean fluently if they are native speakers. This enables even non-experts (e.g., government, high school students, or researchers in a different area)  to easily understand your idea and get interested in your work. If they don't understand your work, you can never be a leader in this society. Although all types of communication are important, here, we only practice writing because of practical restrictions.

**Comment by Lee-Janggun** [comment]
Thank you for the detailed explanation professor!
